src/components/CurrentBookCard/CurrentBookCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link } from 'react-router-dom';
+import ArrowForwardIosRoundedIcon from '@mui/icons-material/ArrowForwardIosRounded';
+
+import { Book } from '../../../types/types';
+import style from './CurrentBookCard.module.scss';
+
+type CurrentBookProps = {
+  books: Book[];
+};
+
+export const CurrentBookCard: React.FC<CurrentBookProps> = ({ books }) => {
+  const currentBook = books.find((book) => book.isCurrentlyReading);
+
+  return (
+    <div>
+      {currentBook && (
```

Comment:
you can do an early return here
```suggestion

if(!currentBook) {
 return null
 }

  return (
    <div>
```